install-build-deps.sh

```diff
@@ -8,22 +8,23 @@ fi
 
 if [ "$(uname)" = "Darwin" ] ; then
   OSARCH="darwin-amd64"
+  env CGO_ENABLED=1 go install -tags extended github.com/gohugoio/hugo
 else
   OSARCH="linux-amd64"
+  env CGO_ENABLED=1 go install -tags extended -ldflags "-linkmode external -extldflags -static" github.com/gohugoio/hugo
```

Comment:
Please factor out the duplicated `go install` and move it out of the `if`. You can use a variable for the differing arguments.

Reply:
Sure, will also double check if linker flags are really needed on Linux, I copied this from somewhere.

Reply:
Ah, you are right. My bad